controlplane/api/v1alpha1/rke2controlplane_types.go

```diff
@@ -37,7 +37,7 @@ type RKE2ControlPlaneSpec struct {
 	Replicas *int32 `json:"replicas,omitempty"`
 
 	// RKE2AgentConfig references fields from the Agent Configuration in the Bootstrap Provider because an RKE2 Server node also has an agent
```

Comment:
Comment is out of sync.

From a stylist point, its good to move the `inline` fields to the start of the struct definition.

Reply:
moved the `inline` field up and updated comments.